apps/ui/src/composables/useAlias.ts

```diff
@@ -0,0 +1,54 @@
+import { Wallet } from '@ethersproject/wallet';
+import { getDefaultProvider } from '@ethersproject/providers';
+import { enabledNetworks, getNetwork, offchainNetworks } from '@/networks';
+import pkg from '../../package.json';
+
+const aliases = useStorage(`${pkg.name}.aliases`, {} as Record<string, string>);
+
+const networkId = offchainNetworks.filter(network => enabledNetworks.includes(network))[0];
+const network = getNetwork(networkId);
+
+export function useAlias() {
+  const { web3 } = useWeb3();
+
+  const wallet = computed(() => {
+    const provider = getDefaultProvider();
+    const privateKey = aliases.value[web3.value.account];
+
+    if (!privateKey) return null;
+
+    return new Wallet(privateKey, provider);
+  });
```

Comment:
Personally I'd just one (public) method from this hook like `getAliasWallet` what would:
1. check storage cache
2. check existing alias
3. create new one if expired

so every consumer has this one call to get it, right now every consumer has to manually call `isValid`/`create` otherwise `wallet` might be `null` which is not great to depend on this hidden functionality via side effects.